finetune/model.py

```diff
@@ -132,11 +142,17 @@ def _model_fn(features, labels, mode, params):
         train = estimator_mode == tf.estimator.ModeKeys.TRAIN
         X = features["tokens"]
         context = features.get("context", None)
-        task_id = features.get("task_id", None)
         Y = labels
         pred_op = None
 
-        with tf.variable_scope(tf.get_variable_scope()):
+        if estimator_mode == tf.estimator.ModeKeys.PREDICT and params.float_16_predict:
```

Comment:
Only request would be factoring this out into a separate fn to keep _model_fn relatively clean